### PR TITLE
test: repeated volume rebuilds should succeed

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Added two rebuild tests which are ignored as they're meant to aid in testing io-engine rebuilds.

Example running with custom io-engine:

export IO_ENGINE_BIN=/home/tiago/git/mayastor/io-engine/target/debug/io-engine
cargo test -p io-engine-tests repeated_volume_double_rebuilds -- --ignored

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: tests/io-engine/tests/rebuild.rs [hunk 1]
```diff
@@ -237,3 +241,813 @@ async fn loopback_nvmf() {
...
+            // let out = std::process::Command::new("sudo")
+            //     .args(vec![
+            //         "dd",
+            //         "if=/dev/nvme2n1".to_string().as_str(),
+            //         "of=0.img".to_string().as_str(),
+            //         "bs=512k",
+            //     ])
+            //     .output()
+            //     .unwrap();
+            // assert!(out.status.success());
```
Comment (Abhinandan-Purkait) [inline tests/io-engine/tests/rebuild.rs]: Do we need this?

File: tests/io-engine/tests/rebuild.rs [hunk 2]
```diff
@@ -237,3 +241,813 @@ async fn loopback_nvmf() {
...
+            //
+            //     cluster
+            //         .composer()
+            //         .exec(
+            //             cluster.csi_container(0).as_str(),
+            //             vec!["nvme", "disconnect-all"],
+            //         )
+            //         .await
+            //         .unwrap();
+            // }
```
Comment (Abhinandan-Purkait) [inline tests/io-engine/tests/rebuild.rs]: Do we need this?

**Review by Abhinandan-Purkait — APPROVED** [review]

File: tests/io-engine/tests/rebuild.rs (hunk 2 above)
Comment (tiagolobocastro) [reply tests/io-engine/tests/rebuild.rs]: No, I'll remove.

File: tests/io-engine/tests/rebuild.rs (hunk 1 above)
Comment (tiagolobocastro) [reply tests/io-engine/tests/rebuild.rs]: Could be useful, I'll add a bool to configure and disable by default.

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**